kubernetes/samples/scripts/create-weblogic-domain-on-azure-kubernetes-service/model-in-image/mii-initial.yaml

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2021, Oracle Corporation and/or its affiliates.
```

Comment:
This looks extremely similar to the current MII sample's model YAML. We try to keep the sample robust, testable, and 'DRY'. 
Can you do one of the following?
- Remove this file, and instead provide instructions for a manual step that copies the existing MII sample and modifies it as needed.
- Update the YAML generator that generates the MII sample model files to accomodate this file (generate a version of it in /tmp/), AND update the related integration test so that it verifies the checked-in generated files match the core MII sample template.  (See my other comments for README instructions about the generator. Note that that the generator script also serves double duty as the integration test.)

Reply:
Will do.